README.rst

```diff
@@ -459,12 +459,12 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
    * - ``protobuf_runtime_directory``
      - ``runtime``
      - Runtime directory for the ``protoc`` protobuf schema parser and code generator
-   * - ``name_strategy``
-     - ``topic_name``
-     - Name strategy to use when storing schemas from the kafka rest proxy service
+   * - ``default_name_strategy``
```

Comment:
yep, I've decided to get rid of this change. This pr needs to be rebased on top of the #754. Probably I will close this pr since the design of making stateful the request for publish using the rest endpoint isn't a great idea.
It's probably mark this pr as draft